src/routes/ConsortiumManager/settings/authorization-policies/AuthorizationPoliciesSettings.js

```diff
@@ -0,0 +1,43 @@
+import { useMemo } from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import AuthorizationPolicies from '@folio/authorization-policies';
```

Comment:
don't import module in another module, for this purpose `stripes-authorization-components` was created